src/v/crypto/tests/BUILD

```diff
@@ -84,3 +86,133 @@ redpanda_cc_gtest(
         "@googletest//:gtest",
     ],
 )
+
+expand_template(
+    name = "openssl_conf",
+    out = "openssl_conf.cnf",
+    substitutions = {
+        "@REDPANDA_DEPS_INSTALL_DIR@": "src/v/crypto/tests",
+    },
+    template = "test_openssl_conf.cnf.in",
+)
+
+select_file(
+    name = "fipsmodule_module",
+    srcs = "@openssl-fips",
+    subpath = "lib/ossl-modules/fips.so",
+)
+
+filegroup(
+    name = "openssl-fips-gen-dir",
+    srcs = ["@openssl-fips"],
+    output_group = "gen_dir",
+)
+
+genrule(
+    name = "fipsmodule_cnf",
```

Comment:
Why does this need to be copied? Can we remove it?

If we cannot remove it: 
1. can we move this to the openssl-fips BUILD file
2. Can we switch to https://github.com/bazelbuild/bazel-skylib/blob/main/docs/copy_file_doc.md

Reply:
Pretty sure we need to copy it, since the file we want a) is not itself exposable as a target from the configure_make rule and b) _must_ be situation in a subdirectory of CWD of a running test. 

> 1. can we move this to the openssl-fips BUILD file

Maybe. I tried a bit, but it wasn't obvious how to reach down into `@openssl-fips` from here.

> 2. Can we switch to `copy_file`

I don't think so, since the file we want isn't actually in the file list of `openssl-fips-gen-dir` or any other target. All we have is the directory, so we need to reach down into it manually (I think).

Reply:
Don't think we can move this one because we still need to copy the config file into the install tree of `crypto/tests`.

Reply:
>Don't think we can move this one because we still need to copy the config file into the install tree of crypto/tests.

would this become a runfile artifact?

Reply:
You mean after the application of the genrule where it currently sits or if we moved it closer to openssl-fips?